tests/index.ts

```diff
@@ -0,0 +1,61 @@
+import * as fs from "fs/promises";
+import Ajv from "ajv";
+import * as YAML from "yaml";
+
+const ajv = new Ajv();
+const verbose = process.argv.includes("--verbose") || process.argv.includes("-v");
+
+type Test = {
+  name: string;
+  description?: string;
+  valid: boolean;
+  schema: string;
+  version: number;
+  data: Object;
+};
+
```

Comment:
Let's just define the TestCase here:

```typescript
const testcases: Test[] = [{
  name: "Single Micro",
  description: "Tests a Spacefile configuration with a single Micro",
  valid: true,
  schema: "spacefile",
  version: 0,
  path: ...,
}]
```

Reply:
I will make a `tests.ts` file 👍

Reply:
Done in 21980a9ab337356a371d16871242c37e0645f832